compiler/noirc_frontend/src/node_interner.rs

```diff
@@ -441,6 +459,12 @@ impl GlobalId {
     }
 }
 
+impl Into<usize> for GlobalId {
+    fn into(self) -> usize {
+        self.0
+    }
+}
```

Comment:
Do you mind changing the `Into` impls into `as_usize()` functions or similar?

Also curious what all these into usize impls are for

Reply:
Great catch. These are artefacts of older methods of doing things that I clearly missed removing when I was checking through our changes. The only one remaining (which is now an `as_index` is used for some iteration-based searching for modules.